rauc_hawkbit/ddi/client.py

```diff
@@ -163,9 +164,9 @@ def build_api_url(self, api_path):
                     **kwargs))
 
         self.logger.debug('GET {}'.format(url))
-        with aiohttp.Timeout(self.timeout):
-            async with self.session.get(url, headers=get_headers,
-                                        params=query_params) as resp:
+        async with self.session.get(url, headers=get_headers,
+                                    params=query_params,
+                                    timeout=self.timeout) as resp:
```

Comment:
It was two indentation levels before and is only one now, thus please also fix indention of the block below